3.14.py

```diff
@@ -0,0 +1,88 @@
+                #Exercise 1 
+
+def right_justify(s):
+    while len(s)!=70:
+        s=' '+s
```

Comment:
there is neatpick in python, that allow to do `s = ' '*69 + s`

Reply:
then the last character is not obtained № 70 (

Reply:
Done!